doc/reference_accessor.xml

```diff
@@ -382,8 +774,7 @@ SELECT ST_AsEWKT(ST_EndPoint('LINESTRING(1 1 2, 1 2 3, 0 0 5)'));
 		  <refnamediv>
 			<refname>ST_Envelope</refname>
 
-			<refpurpose>Returns a geometry representing the double precision (float8) bounding box of the
-			supplied geometry.</refpurpose>
+			<refpurpose>Returns a geometry representing the double precision (float8) bounding box of a geometry.</refpurpose>
```

Comment:
float8 is confusing to newbies. "is it less precise than usual or more precise than usual".

Reply:
Fixed in comment.  Will leave the longer more accurate description in the main text.  Newbies can look it up :)